### PR TITLE
add a latest minor version arn export to outputs (i.e. 1.1.x)

### PR DESCRIPTION
**Pull request by dgoodellrhy**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -14,3 +14,12 @@ output "component_arn" {
     aws_cloudformation_stack.this
   ]
 }
+
+output "latest_minor_version_arn" {
+  description = "ARN of the EC2 Image Builder Component"
+  value       = "arn:aws:imagebuilder:${local.region}:${local.account_id}:component/${lower(var.name)}/${local.latest_component_minor_version}"
+
+  depends_on = [
```
Comment (sblack4) [inline outputs.tf:22]: Have you tested this? It's gotten me in trouble before

**Review by sblack4 — DISMISSED** [review]

File: outputs.tf (hunk 1 above)
Comment (dgoodellrhy) [reply outputs.tf:22]: y'know that's a good call.  This is based off of similar work already done by Scott but I haven't tested my variant, I'll give that a go

File: bin/install-ubuntu.sh [hunk 2]
```diff
@@ -19,5 +19,5 @@ git config --global init.templateDir ~/.git-template
 pre-commit init-templatedir ~/.git-template
 
 echo 'installing terraform with tfenv'
-tfenv install min-required
-tfenv use min-required
+tfenv install latest:^0.12
+tfenv use latest:^0.12
```
Comment (sblack4) [inline bin/install-ubuntu.sh:23]: `tfenv` is getting a little too crazy with their commands.

**Review by sblack4 — APPROVED** [review]
:shipit:

**Comment by dgoodellrhy** [comment]
confirmed this works like I expect it to, merging

**Pull request merged**